src/refapp-grid/refapp-grid.css

```diff
@@ -14,3 +14,8 @@
 .no-rows-text {
   margin: 5px;
 }
+
+.table [role="columnheader"] {
+  background-color: bisque;
```

Comment:
Change the to match with existing grid layout. It was the sample colour we chose to check